src/anchoredinterval.jl

```diff
@@ -176,23 +176,11 @@ span(interval::AnchoredInterval{P}) where P = abs(P)
 
 ##### CONVERSION #####
 
-# Allows an interval to be converted to a scalar when the set contained by the interval only
-# contains a single element.
-function Base.convert(::Type{T}, interval::AnchoredInterval{P,T}) where {P,T}
+function Base.only(interval::AnchoredInterval{P}) where {P}
```

Comment:
Need to properly deprecate the use of `convert`. We can turn around and drop this and other depreciations in version 2